<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitEditor</class>
 <widget class="QMainWindow" name="KitEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KitEditor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QRadioButton,QComboBox,QSpinBox,QTableWidget::item,QTableView::item,QCheckBox,QDateTimeEdit{
font-family:'思源黑体 CN Normal';
font-size:12pt;
}

QPushButton,QHeaderView,QGroupBox,QTabBar,QTableWidget{
font-family:'思源黑体 CN Bold';
font-size:14pt;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetTop" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{
	background-color:rgb(223, 234, 255);
	border:1px solid rgb(176, 215, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="styleSheet">
               <string notr="true">QWidget
{
	border:0px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="btOpenFile">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton{
	background-color:rgb(121, 154, 247);
	color:rgb(103, 80, 164);
}</string>
                   </property>
                   <property name="text">
                    <string>打开</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}</string>
                   </property>
                   <property name="text">
                    <string>  路径:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lbPath">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="styleSheet">
               <string notr="true">QWidget
{
	border:0px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="horizontalWidget_2" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}QQQ</string>
                    </property>
                    <property name="text">
                     <string>试剂盒名称: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="editAbbrName">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>4</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>24</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLineEdit{
	border:1px solid rgb(173, 175, 191);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}QQQ</string>
                    </property>
                    <property name="text">
                     <string>  全称: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>4</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLineEdit{
	border:1px solid rgb(173, 175, 191);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}QQQ</string>
                    </property>
                    <property name="text">
                     <string>  反应体系: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEdit_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>1</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLineEdit{
	border:1px solid rgb(173, 175, 191);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="styleSheet">
                     <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}QQQ</string>
                    </property>
                    <property name="text">
                     <string>μL</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}QQQ</string>
                    </property>
                    <property name="text">
                     <string>  温控流程: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_2">
                    <property name="minimumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QComboBox{
	border:1px solid rgb(173, 175, 191);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel{
	color:rgb(114, 118, 196);
}QQQ</string>
                    </property>
                    <property name="text">
                     <string>  机型: </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox">
                    <property name="minimumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QComboBox{
	border:1px solid rgb(173, 175, 191);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>10</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_5">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color:rgb(121, 154, 247);
	color:rgb(103, 80, 164);
}</string>
                    </property>
                    <property name="text">
                     <string>新建</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>10</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_4">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color:rgb(121, 154, 247);
	color:rgb(103, 80, 164);
}</string>
                    </property>
                    <property name="text">
                     <string>校验</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>10</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color:rgb(121, 154, 247);
	color:rgb(103, 80, 164);
}</string>
                    </property>
                    <property name="text">
                     <string>发布</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>10</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_2">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>35</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
	background-color:rgb(121, 154, 247);
	color:rgb(103, 80, 164);
}</string>
                    </property>
                    <property name="text">
                     <string>备份</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetCenter" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	background-color:rgb(223, 234, 255);
	border:1px solid rgb(176, 215, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QTableView" name="mainTable">
            <property name="styleSheet">
             <string notr="true">QTableView
{
	border:0px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetBottom" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget{
	background-color:rgb(223, 234, 255);
	border:1px solid rgb(176, 215, 255);
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>思源黑体 CN Bold</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color:rgb(114, 118, 196);
	font-size:18px;
	font-family:&quot;思源黑体 CN Bold&quot;;
}</string>
              </property>
              <property name="text">
               <string>质控</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="positiveTable">
              <property name="styleSheet">
               <string notr="true">QTableView
{
	border:0px;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
